src/js/shield_defs.js

```diff
@@ -632,15 +632,12 @@ function banneredShield(baseDef, modifiers) {
   };
 }
 
-export function loadShields(shieldImages) {
+export function loadShields() {
```

Comment:
A couple of the scripts rely on passing in mocked versions of `shieldImages`. They no longer need to pass anything in, but instead they should map `spriteBlank` to a full image URL afterwards.

https://github.com/ZeLonewolf/openstreetmap-americana/blob/d35b7e3629ad9cf57f65183eb7666c21d2aea134/scripts/taginfo.js#L14-L20 https://github.com/ZeLonewolf/openstreetmap-americana/blob/d35b7e3629ad9cf57f65183eb7666c21d2aea134/scripts/status_map.js#L25-L31

Reply:
> instead they should map spriteBlank to a full image URL afterwards.

Can you elaborate?

Reply:
Somehow or another, these scripts need to end up with absolute URLs to shield images. Previously, they relied on a bit of a hack where `loadShields()` would happen to stick absolute URLs in each definition because the mock “sprite sheet” consisted of absolute URLs. But now we can just insert some code after the call to `loadShields()` that goes through each entry in the returned object, replacing its `spriteBlank` with an absolute URL.

Reply:
Okay, I think we have a working solution in 0e32599.

Reply:
In fact, I just discovered that we don't need to parse the sprite JSON at all, so I scrubbed that in cdbef18